types/slack-mock/index.d.ts

```diff
@@ -1,8 +1,9 @@
 // Type definitions for slack-mock 1.1
 // Project: https://github.com/Skellington-Closet/slack-mock
 // Definitions by: Kris Kalavantavanich <https://github.com/kkalavantavanich>
+//                 Florian Keller <https://github.com/ffflorian>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.4
+// TypeScript Version: 3.4
```

Comment:
Why did you have to upgrade the TypeScript dependency? I didn't see anything that would require it.

Reply:
Because the (now extrnal) nock types require TypeScript >= 3.4. You can test it locally by running `npm run lint slack-mock` when leaving the TypeScript version at 2.4.

Reply:
Oh, got it. Thank you.